src/SIL.XForge.Scripture/ClientApp/src/app/change-password/change-password.component.ts

```diff
@@ -0,0 +1,60 @@
+import { HttpClient, HttpHeaders } from '@angular/common/http';
```

Comment:
This component should be moved to the `src/xforge-common` folder, since it will be shared between Language Forge and Scripture Forge.